packages/open-ui/src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,109 @@
+@import '../../styles/index.scss';
+
+$z-index: 999;
+
+.overlay {
+  background: $white;
+  bottom: 0;
+  cursor: alias;
+  left: 0;
+  opacity: 0.3;
+  position: fixed;
+  right: 0;
+  top: 0;
+  visibility: hidden;
+  z-index: $z-index;
+
+  &.active {
+    visibility: visible;
+  }
+}
+
+.modalWrapper {
+  left: 50%;
+  max-width: 600px;
+  padding: 2 * $vr;
+  opacity: 0;
+  position: fixed;
+  top: calc(50% - 12px);
+  transform: translate(-50%, -50%);
+  transition: all 0.4s ease;
+  width: 100%;
+  z-index: $z-index + 1;
+
+  &.active {
+    opacity: 1;
+    top: 50%;
+  }
+}
+
+.modal {
+  background: $off-black;
+  border-radius: $vr;
+  border: 3px solid $primary-color;
+  box-shadow: $base-shadow;
+  max-height: 90vh;
+  position: relative; // for close button
+  overflow-y: scroll;
+  z-index: $z-index;
+}
+
+.titleWrapper {
+  background: darken($off-black, 2%);
+  border-top-left-radius: $vr;
+  border-top-right-radius: $vr;
+  padding: 3 * $vr;
+}
+
+.content {
+  padding: 3 * $vr;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: column-reverse;
+  flex-wrap: wrap;
+  padding: 0 (3 * $vr) (3 * $vr) (3 * $vr);
+
+  button {
+    flex-basis: 100%;
+
+    &:nth-of-type(2) {
+      margin-bottom: $vr;
+    }
+  }
+
+  @media #{$small-up} {
+    flex-wrap: no-wrap;
+    flex-direction: row;
+    justify-content: flex-end;
+
+    button {
+      flex-basis: auto;
+
+      &:nth-of-type(2) {
+        margin-bottom: 0;
+        margin-left: 2 * $vr;
+      }
+    }
+  }
+}
+
+// Intent colours
+.modalWrapper.intentSuccess {
```

Comment:
@remotealex you should add this under the modalWrapper class above, like currently we are recreating it

Reply:
I prefer to have these custom ones at the bottom. Nest SCSS doesnt actually result in less CSS so I think it's okay like this.